src/components/molecules/KeyValue/KeyValue.stories.tsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import { Meta, StoryObj } from "@storybook/react";
+
+// Components
+import Pill from "@components/atoms/Pill";
+import TextLink from "@components/atoms/TextLink";
+import KeyValue, { IKeyValueProps } from "@components/molecules/KeyValue";
+import Key from "@components/molecules/KeyValue/Key";
+import Value from "@components/molecules/KeyValue/Value";
+
+// Helpers
+import { args, propCategory } from "../../../../stories/assets/storybook.globals";
+
+const key = <Key infoText="Info text">Title</Key>;
+
+const meta: Meta<typeof KeyValue> = {
+    title: "Molecules/KeyValue",
+    component: KeyValue,
```

Comment:
```suggestion
    component: KeyValue,
    subcomponents: { Value },
```

Add subcomponent to show documentation for Value component.